diff --git a/dist/index.mjs b/dist/index.mjs
index 1850cd61394b63c67b6301ff46977d771458bd9c..a667a7f8d387645f74f3b106bb0af8e715c61e6b 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -244,11 +244,11 @@ var SSRStylesPlugin = class {
 			if (this.isPublicAsset(stripped, nitro)) {
 				needsPublicAsset = true;
 				changed = true;
-				out += "${publicAssetsURL(" + JSON.stringify(rawUrl) + ")}";
+				out += "url(" + (match[1] || "") + "${publicAssetsURL(" + JSON.stringify(rawUrl) + ")}" + (match[1] || "") + ")";
 			} else if (this.isBuildAsset(stripped)) {
 				needsBuildAsset = true;
 				changed = true;
-				out += "${buildAssetsURL(" + JSON.stringify(rawUrl) + ")}";
+				out += "url(" + (match[1] || "") + "${buildAssetsURL(" + JSON.stringify(rawUrl) + ")}" + (match[1] || "") + ")";
 			} else out += this.escapeTemplateLiteral(full);
 			lastIndex = index + full.length;
 		}
